### PR TITLE
Added alternative for ImageReflectionView with enhanced ReflectionVie…

### PR DESCRIPTION
**Pull request by snehilrx**
Implemented a custom linear layout to show reflection of the view contained in it. 
Feature 👍🏾 
10x faster load time than  ReflectionImageView
Matches the view draw in real time.

### CODE REVIEW

**Comment by sparrow007** [comment]
@snehilrx Thank you for the pull request but before merge into master please add the package name at the top of the file because if that's not added it keeps generating the error of inflation.

File: carouselrecyclerview/src/main/java/com/jackandphantom/carouselrecyclerview/view/ReflectionViewContainer.kt
Comment (sparrow007) [inline carouselrecyclerview/src/main/java/com/jackandphantom/carouselrecyclerview/view/ReflectionViewContainer.kt:106]: Hi, @snehilrx  I don't see this styleable in attrs.xml, so which style you are using here?

**Review by sparrow007 — CHANGES_REQUESTED** [review]

**Comment by snehilrx** [comment]
@sparrow007 I had fixed all the issue as mentioned by you and also the issues that I had found.

Here is a sample usage
```xml
<?xml version="1.0" encoding="utf-8"?>
<com.jackandphantom.carouselrecyclerview.view.ReflectionViewContainer
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    xmlns:android="http://schemas.android.com/apk/res/android"
    app:reflect_relativeDepth="0.5"
    app:reflect_gap="0dp"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    >
    <ImageView
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:src="@drawable/guypro"
        android:scaleType="fitXY"
        android:id="@+id/image" />
</com.jackandphantom.carouselrecyclerview.view.ReflectionViewContainer>
```


<p>

</p>

Sample Screen

**Comment by sparrow007** [comment]
Yeah I see and I love what you do (awesome work man), but if you see your files changes currently which have some error because of different commits (as I mentioned above) so I can't merge it would you please create another pull with all new changes so that it will very easy for me to merge.

**Comment by snehilrx** [comment]
Going to sleep. Goodnight. I will clarify things tomorrow.

**Comment by sparrow007** [comment]
Yeah, no problem :)

**Comment by snehilrx** [comment]
Sorry for the delayed fix, I was busy with some work. I think it should work now.

**Comment by sparrow007** [comment]
No problem, thanks i am reviewing the codes now :)

**Comment by sparrow007** [comment]
LGTM

**Review by sparrow007 — APPROVED** [review]

**Pull request merged**
